parsedatetime/tests/TestRussianLocale.py

```diff
@@ -78,6 +78,24 @@ def testDates(self):
         self.assertExpectedResult(self.cal.parse('25.8', start), (target, 1))
         self.assertExpectedResult(self.cal.parse('25.08', start), (target, 1))
 
+    def testDatesLang(self):
+        if sys.version_info >= (3, 0):
+            target = datetime.datetime(2006, 8, 25, 23, 5).timetuple()
```

Comment:
Yeah but it shouldn't be inconsistent between PY2x and PY3K. We'd better figure it out why it fails when using Python 2.x.